ui/src/app/views/workspace-edit/component.tsx

```diff
@@ -596,8 +596,8 @@ export class WorkspaceEditComponent extends ReactWrapperBase {
     this._location.back();
   }
 
-  reloadWorkspace(namespace, id): void {
-    this.workspaceStorage.reloadWorkspace(namespace, id);
+  reloadWorkspace(namespace, id): Promise<WorkspaceData> {
+    return this.workspaceStorage.reloadWorkspace(namespace, id);
```

Comment:
I don't see anyone consuming this, which leads me to believe this is either not necessary or it has something to do with getting Angular to detect the promise. If it's the latter, please add a comment explaining.

Reply:
It has to do with allowing the workspace edit component to properly await the completion of `reloadWorkspace` so it doesn't preload the stale one. 

Specifically, this bug was caused by this line here: 
https://github.com/all-of-us/workbench/pull/2104/files#diff-6bc93013088145c1dbc94c5714d63f39R363

We wanted to block navigation on the update finishing, but since we weren't actually handing down the promise, it wasn't functioning as intended.

Reply:
Updating to be promise<void> though, because the actual object returned was not consumed.

Reply:
Resolving now that we are using that object to publish to the store.

Reply:
opt: I don't really need to code comment here anymore - I think it's clear given the new code that it wants to wait on the response. IN the initial PR the return value wasn't actually used by anything.